scripts/vim-patch.sh

```diff
@@ -132,37 +132,54 @@ get_vim_patch() {
 
   # Patch surgery: preprocess the patch.
   #   - transform src/ paths to src/nvim/
-  local vim_full="$(git show -1 --pretty=medium "${vim_commit}" \
+  local vim_full
+  vim_full="$(git show -1 --pretty=medium "${vim_commit}" \
     | LC_ALL=C sed -e 's/\( [ab]\/src\)/\1\/nvim/g')"
   local neovim_branch="${BRANCH_PREFIX}${vim_version}"
 
   cd "${NEOVIM_SOURCE_DIR}"
-  local git_remote=$(find_git_remote)
-  local checked_out_branch="$(git rev-parse --abbrev-ref HEAD)"
+  local git_remote
+  git_remote=$(find_git_remote)
+  local checked_out_branch
+  checked_out_branch="$(git rev-parse --abbrev-ref HEAD)"
 
   if [[ "${checked_out_branch}" == ${BRANCH_PREFIX}* ]]; then
     echo "✔ Current branch '${checked_out_branch}' seems to be a vim-patch"
     echo "  branch; not creating a new branch."
   else
     echo
     echo "Fetching '${git_remote}/master'."
-    output="$(git fetch "${git_remote}" master 2>&1)" &&
-      echo "✔ ${output}" ||
+    output="$(git fetch "${git_remote}" master 2>&1)"
+    if [[ $output ]]
+    then
+      echo "✔ ${output}"
+    else
       (echo "✘ ${output}"; false)
+    fi
 
     echo
     echo "Creating new branch '${neovim_branch}' based on '${git_remote}/master'."
     cd "${NEOVIM_SOURCE_DIR}"
-    output="$(git checkout -b "${neovim_branch}" "${git_remote}/master" 2>&1)" &&
-      echo "✔ ${output}" ||
+
+
+    output="$(git checkout -b "${neovim_branch}" "${git_remote}/master" 2>&1)"
+    if [[ $output ]]
+    then
+      echo "✔ ${output}"
+    else
       (echo "✘ ${output}"; false)
```

Comment:
Yes, the `set -e` is what requires this.  This change now means that if `git checkout` errors, the script immediately exits instead of chaining through to the `(echo "✘ ${output}"; false)`.